templates/ingester/ingester-dep.yaml

```
@@ -81,8 +81,10 @@ spec:
             - name: gossip
               containerPort: {{ .Values.config.memberlist.bind_port }}
               protocol: TCP
+          {{- if .Values.ingester.startupProbe }}
           startupProbe:
             {{- toYaml .Values.ingester.startupProbe | nindent 12 }}
+          {{- end}}
```

```suggestion
          {{- end }}
```

why do you hate spaces 😆

---

Sorry, I think I must have been copying and pasting blocks with missing whitespace. I fixed this and other whitespace throughout.